### PR TITLE
reorganize generalsettings, closes #2380 and fixes #2293

### LINKED ISSUE #2293: 'Light icon' should be able to be checked even when 'Show system tray icon' is unchecked
capture2

### LINKED ISSUE #2380: Use indentation in qTox Settings
Hello,

another minor issue:
Use identation in the qTox Settings to improve readability and clarity.

So, I would indent 'Open each chat in an individual window' because it is a subitem of 'Multiple windows mode'. 
Screenshot

### PR DESCRIPTION
**Pull request by agilob**
- tab order on widgets
- better spacing
- consistent text
- indentation for similar settings

weblate14

### CODE REVIEW

**Comment by agilob** [comment]
closes #2380 and fixes #2293

**Comment by agilob** [comment]
Someone broke travis and it's not me.

**Comment by zetok** [comment]
> Someone broke travis and it's not me.

yeah, it sometimes randomly fails on getting dependencies; restarted

**Comment by ovalseven8** [comment]
Good work! @agilob 
But perhaps you can make the indentation only three or four spaces. In my mind it's too wide on the right side.

**Comment by agilob** [comment]
Indentation is automatic and its size will be different in different languages, second column starts where first ends. That's how other parts of settings work in qtox so I followed current design.

**Comment by SkyzohKey** [comment]
This is an ugly way to display settings. :x

**Comment by ProMcTagonist** [comment]
Far too indented.

**Comment by Zer0-One** [comment]
yeah, i'm not digging the indented style

**Comment by agilob** [comment]
Second approach. Here I also changed order of items, autostart it on the top and nested group was removed because it was ugly and I didnt like it.

weblate15

File: src/widget/form/settings/generalform.cpp [hunk 1]
```diff
@@ -151,12 +151,12 @@ GeneralForm::GeneralForm(SettingsWidget *myParent) :
     //general
     connect(bodyUI->checkUpdates, &QCheckBox::stateChanged, this, &GeneralForm::onCheckUpdateChanged);
     connect(bodyUI->transComboBox, SIGNAL(currentIndexChanged(int)), this, SLOT(onTranslationUpdated()));
-    connect(bodyUI->cbAutorun, &QCheckBox::stateChanged, this, &GeneralForm::onAutorunUpdated);
+    connect(bodyUI->cbAutorun, &QCheckBox::stateChanged, this, &GeneralForm::onAutorunUpdated);\
```
Comment (zetok) [inline src/widget/form/settings/generalform.cpp]: `\` ?

File: src/widget/form/settings/generalsettings.ui [hunk 2]
```diff
@@ -38,9 +38,9 @@
       <property name="geometry">
        <rect>
         <x>0</x>
-        <y>0</y>
-        <width>639</width>
-        <height>1388</height>
+        <y>-180</y>
+        <width>631</width>
+        <height>1358</height>
```
Comment (zetok) [inline src/widget/form/settings/generalsettings.ui]: Is this change really needed?

File: src/widget/form/settings/generalform.cpp (hunk 1 above)
Comment (agilob) [inline src/widget/form/settings/generalform.cpp]: weblate16
:D

File: src/widget/form/settings/generalsettings.ui (hunk 2 above)
Comment (agilob) [inline src/widget/form/settings/generalsettings.ui]: This is automated change from qtcreator, I guess.

**Comment by SkyzohKey** [comment]
Maybe instead of changing this you should make a mockup to better display the settings view :x

**Comment by agilob** [comment]
I dont know how to make mockup, I can make a .ui in few minutes.

Here, have my old blog banner:

logo

**Comment by zetok** [comment]
PR seems to be mostly ok - there is small thing to fix, so @agilob you've got a PR to merge: https://github.com/agilob/qTox/pull/3 :P

I didn't test autostart and network settings.
